CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.362 Mar 06 2024
+- Changed `UserName` attribute for TokenClaimMappings to `Username`.
+
```

Comment:
These changes are typically done on a separate MR once the main PR to introduce the changes has been merged; https://github.com/openshift-online/ocm-sdk-go/blob/main/CONTRIBUTING.md#releasing-a-new-ocm-api-model-version

Reply:
Thanks @machi1990 , will remove this commit